container/ubuntu16_04/layers/bazel/BUILD

```diff
@@ -20,13 +20,16 @@ load("//container/rules:docker_toolchains.bzl", "language_tool_layer")
 load("//container/rules:docker_toolchains.bzl", "toolchain_container")
 load("@base_images_docker//util:run.bzl", "container_run_and_extract")
 load(
-    "//container/ubuntu16_04/layers/bazel:version.bzl",
+    "//container/common/bazel:version.bzl",
     "BAZEL_VERSION_SHA256S",
     "BAZEL_VERSION_TO_CONFIG_VERSION",
 )
 
 exports_files(glob(["*.yaml"]))
 
+# Used in substitution in bazel.yaml.tpl and bazel.bazelrc.tpl
+CONFIG_BASE = "ubuntu16_04_clang/1.0"
```

Comment:
same, move further up the directory structure

Reply:
Done.